components/gsma-authenticators/src/main/java/com/wso2telco/gsma/authenticators/voice/ValidSoftJsonHelper.java

```diff
@@ -0,0 +1,88 @@
+package com.wso2telco.gsma.authenticators.voice;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.codehaus.jettison.json.JSONException;
+import org.codehaus.jettison.json.JSONObject;
+
+/**
+ * Created by sheshan on 5/4/17.
+ */
+
+public class ValidSoftJsonHelper {
+
+    private String isUserActiveRequestJson;
+    private String userRegistrationAndAuthenticationJson;
+    private final String UNKNOWN_USER = "UNKNOWN_USER";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.wso2telco.com:9000/coding_rules#rule_key=squid%3AS1170)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "UNKNOWN_USER" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.wso2telco.com:9000/coding_rules#rule_key=squid%3AS00116)